projects/wave-core-new/src/lib/datasets/add-data/add-data.component.ts

```diff
@@ -38,7 +39,7 @@ export class AddDataComponent implements OnInit {
         this.layoutService.setSidenavContentComponent(sidenavConfig);
     }
 
-    static createDataSetListButton(): AddDataListButton {
+    static createDataSetListButton(): AddDataButton {
```

Comment:
maybe we can use the momentum and say `Datasets`? :wink:

Reply:
yeah I will do a PR at some point unifying it across front and backend to Dataset